rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadio.java

```diff
@@ -1,8 +1,11 @@
 package com.polidea.rxandroidble.internal;
 
 import rx.Observable;
+import rx.Scheduler;
 
 public interface RxBleRadio {
 
+    Scheduler scheduler();
```

Comment:
This was done mainly so that custom operation can access the scheduler (via the builder). The `LongWriteOperationBuilderImpl` now also use `RxBleRadio#scheduler` instead of passing around `AndroidSchedulers.mainThread()`.